core/src/test/java/io/cucumber/core/plugin/PluginFactoryTest.java

```diff
@@ -49,12 +49,16 @@
     void instantiates_junit_plugin_with_file_arg() {
         Object plugin = fc.create(parse("junit:" + tmp.resolve("cucumber.xml")));
         assertThat(plugin.getClass(), is(equalTo(JUnitFormatter.class)));
+
+        release_resources(plugin);
```

Comment:
Please use camelCase - we only use snake_case for test methods (because it reads better in test output).